### ISSUE TITLE
Groups are not created at user creation

### ISSUE THREAD

**Issue by pthoelken** [EsperoTech/yaade#86]
Take a look into the video to checkout the bug. The groups are not created, when I create a user. 
https://github.com/EsperoTech/yaade/assets/12643853/52dd57ac-38b9-4843-bfd4-0dee4be01f14

**Comment by jonrosner** [comment]
Yeah good point, you would have to hit space so that it recognizes it as a group but the UX would be better if it also takes into account what is currently in the input field as a new group. I'll fix it in the next update.

**Comment by pavelpp-topia** [comment]
@jonrosner looks like it's still not fixed, I came here with the same issue. Hitting space works as a workaround

**Comment by madeeldevops** [comment]
Same Issue. Is it fixed yet?

**Comment by jonrosner** [comment]
No, this was not prioritized yet.
